genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -100,6 +100,14 @@ rights for this to work.
 to work.
 |false
 
+|genie.jobs.users.activeJobsLimitEnabled
```

Comment:
you might want to change this to something like `genie.jobs.users.activeLimit.enabled` and then the other can be named `genie.jobs.users.activeLimit.count` that way it's easier to group them. Just a thought.